tests/contest/contest/src/tests/uid_mappings/uid_mappings_test.rs

```diff
@@ -0,0 +1,66 @@
+use std::vec;
+
+use oci_spec::runtime::{
+    LinuxBuilder, LinuxIdMapping, LinuxIdMappingBuilder, LinuxNamespace, LinuxNamespaceBuilder,
+    LinuxNamespaceType, ProcessBuilder, Spec, SpecBuilder,
+};
+use test_framework::{Test, TestGroup, TestResult};
+
+use crate::utils::test_inside_container;
+use crate::utils::test_utils::CreateOptions;
+
+fn create_spec(uid_mappings: Vec<LinuxIdMapping>, gid_mappings: Vec<LinuxIdMapping>) -> Spec {
+    // Get default namespaces and add user namespaces
```

Comment:
nit: We don't need this comment because this behavior is obvious from the source code.